server/controller/usercontroller.js

```diff
@@ -0,0 +1,2 @@
+import User from '../model/user'
+
```

Comment:
@otitoju  let's add a dummy controller to return a dummy response message.

Reply:
Alright